### ISSUE TITLE
(since v0.6.9) property declared multiple times error, even if allowed

### ISSUE THREAD

**Issue by jlsjonas** [VladRassokhin/intellij-hcl#122]
Thank you for opening an issue.
In this template paragraph text could be removed, however please retain headers.

### Prerequisites
 - [x] Ensure you have latest version of plugin installed
 - [x] Search for possible issue duplicates

### Installation details
 - [ ] IDE version (`Help->About->Copy to Clipboard`)
WebStorm 2017.2
Build #WS-172.3317.70, built on July 14, 2017
Licensed to Starcoders EOOD / Jonas De Kegel
You have a perpetual fallback license for this version
Subscription is active until September 19, 2018
JRE: 1.8.0_152-release-915-b5 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.12.6
 - [ ] intellij-hcl plugin version (`Settings->Plugins`)
0.6.9
 - [ ] Terraform version (`terraform -v`)
0.10.8 (latest homebrew version)

### Terraform Configuration Files
```hcl
resource "aws_security_group" "example" {
  name_prefix = "example-"
  description = "Allow all inbound ssh & http(s) traffic"

  ingress = {
    from_port = 443
    to_port = 443
    protocol = "tcp"
    cidr_blocks = [
      "0.0.0.0/0"]
  }

  ingress = {
    from_port = 80
    to_port = 80
    protocol = "tcp"
    cidr_blocks = [
      "0.0.0.0/0"]
  }

  ingress = {
    from_port = 22
    to_port = 22
    protocol = "tcp"
    cidr_blocks = [
      "0.0.0.0/0"]
  }
}
```

### Expected Behavior
What should have happened?
No errors, as per [official docs](https://www.terraform.io/docs/providers/aws/r/security_group.html#ingress) (+ the fact that terraform itself doesn't complain)

### Actual Behavior
What actually happened?
![image](https://user-images.githubusercontent.com/932193/32956429-07d8921a-cbb9-11e7-813c-31ae3ed07323.png)

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
Have a aws_security_group resource block with multiple ingress blocks

EDIT: note that only the (older-style?) including an `=` are affected

**Comment by VladRassokhin** [comment]
That's new inspection in 0.6.9. Will try to fix it, as workaround remove '=' characters.

**Comment by VladRassokhin** [comment]
I've published 0.6.9.1 with fix to the plugins repository, please try it out.
